ntoskrnl/io/iomgr/volume.c

```diff
@@ -17,6 +17,8 @@
 
 /* GLOBALS ******************************************************************/
 
+#define TAG_DEV2DOS ' d2D'
```

Comment:
There's a reason we have a `tag.h` file to keep pool tags in a centralized place, right? Use that.

Reply:
Even for a tag that is only going to be used twice, in the same file (volume.c), in the whole kernel code?
(I privileged locality of definitions here.)

In addition, few other definitions that weren't added to tag.h for unknown reason:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=define+TAG_
![image](https://github.com/reactos/reactos/assets/1969829/05840059-f0c3-47d7-b7e3-fc8ce4c9a3fe)

Reply:
Hermes' Locality-argument is legit imho, as long as tag is used just in single c file. Even in the shared header you won't have any guarantee that there aren't duplicated strings if that header is 10000 lines long.

Reply:
> the habit to diverge at putting new tags in different places
It's for the same reason that you don't necessarily put all your function declarations / prototypes into the same header, if your functions are private. **What do we gain from putting all of them at a single place? (beside the false hope of minimizing duplicates)** It's increasing the need for including that header artificially, (increasing build-times as well).